build/Dockerfile-xtz

```diff
@@ -0,0 +1,15 @@
+FROM golang:latest as build
+
+WORKDIR $GOPATH
+
```

Comment:
should be WORKDIR $GOPATH/src/

I got this error with push otherwise ?

can't load package: package github.com/diadata-org/diadata/internal/pkg/blockchain-scrapers/blockchains/tezos/scrapers/xtz: no Go files in /go/src/github.com/diadata-org/diadata/internal/pkg/blockchain-scrapers/blockchains/tezos/scrapers/xtz